treeherder/model/models.py

```diff
@@ -548,14 +549,27 @@ def update_after_verification(self, user):
         if not self.is_fully_verified():
             return
 
+        intermittent = 'autoclassified intermittent'
```

Comment:
There's also the `failure_classification` name of `'intermittent'`:
https://github.com/mozilla/treeherder/blob/b880bcd30afa03289f43a79fe193bdcbc12dd192/treeherder/model/fixtures/failure_classification.json#L27

...so maybe a different variable name to `intermittent` might make sense?

Reply:
😂 

TIL, and yes definitely.

Reply:
Fixed and rebased.